manubot/cite/cite_command.py

```diff
@@ -71,8 +72,10 @@ def cli_cite(args):
     # generate CSL JSON data
     csl_list = list()
     for citation in args.citations:
-        citation = standardize_citation(citation)
         try:
+            if not is_valid_citation_string(f'@{citation}'):
+                continue
```

Comment:
Should we log something here, perhaps at the `INFO` level, to report that some of the citation strings were skipped?  I tested
```
$ manubot cite fig:123 eq:56
[]
```

That behavior is valid, but the silent output could be confusing.